ledger/catchpointtracker.go

```diff
@@ -1007,11 +1012,37 @@ func (ct *catchpointTracker) accountsUpdateBalances(accountsDeltas compactAccoun
 		}
 	}
 
-	if accumulatedChanges >= trieAccumulatedChangesFlush {
```

Comment:
Seems so. I think the intent of the code I removed was to be _inside_ the loop, so that it would be committed periodically, but there seemed agreement that we could just remove it rather than put it in there.